app/src/main/res/xml/preferences.xml

```diff
@@ -45,6 +45,12 @@
             android:summary="@string/pref_persistNotify_sum"
             android:title="@string/pref_persistNotify_title"/>
         <CheckBoxPreference
+            android:defaultValue="false"
```

Comment:
what would you think about defaulting to 'true'?  Most other audio apps do this by default (Spotify, Google Play, etc...) so it would make sense.  If people don't like it they can turn it off.

By making it the default the user gets the 'better' experience without having to root around for it.